Source/WebKit/GPUProcess/graphics/RemoteRenderingBackend.cpp

```diff
@@ -415,8 +320,9 @@ void RemoteRenderingBackend::releaseRenderingResource(RenderingResourceIdentifie
 {
     assertIsCurrent(workQueue());
     m_remoteDisplayLists.take(renderingResourceIdentifier);
-    auto success = m_remoteResourceCache.releaseRenderingResource(renderingResourceIdentifier);
-    MESSAGE_CHECK(success, "Resource is being released before being cached.");
+    m_remoteImageBuffers.take(renderingResourceIdentifier);
```

Comment:
Since `RemoteImageBuffer` is more of a first-class IPC object now, should releasing them be a separate message, rather than `releaseRenderingResource`?

Reply:
That would seem to be in order, thanks. Done.
I also fixed an issue that prompted me original to leave the code in place which stores the image buffer also to the resource cache. I removed that code now, the image buffers are stored only to the rrb.